mergify_engine/web.py

```diff
@@ -233,14 +233,19 @@ def PullRequestUrl(v):
             message="Mergify not installed on repository '%s'" % owner
         )
 
+    subscription = asyncio.run(sub_utils.get_subscription(installation["id"]))
+
     with github.get_client(owner, repo, installation) as client:
         try:
             data = client.item(f"pulls/{pull_number}")
         except http.HTTPNotFound:
             raise PullRequestUrlInvalid(message=("Pull request '%s' not found" % v))
 
         return context.Context(
-            client, data, [{"event_type": "mergify-simulator", "data": []}]
+            client,
+            data,
+            subscription,
+            [{"event_type": "mergify-simulator", "data": []}],
```

Comment:
Wondering how it was working before?

Reply:
Sub and sources was missing but unused, so it worked, now it's used so we catch it with our tests.